### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by harrislapiroff** [asmeurer/iterm2-tools#16]
Looks like you've done some work on this since then, but the last release on PyPI seems to be from 2016. Happy to help with packaging if I can!

**Comment by asmeurer** [comment]
Yes, I have made some changes here. Some of the changes are backwards incompatible (API cleanups), which would break existing libraries. I've been unsure if I really want to do that, which is why there hasn't been a release. 

Since then, iterm2 has created its own Python API library. I've discussed with the iTerm2 author, and we agree that it would be best to merge this code into the iterm2 Python library itself (see https://gitlab.com/gnachman/iterm2/issues/7827). So my current plan of action is to do this, and most likely deprecate this library, meaning I won't be releasing any of the backwards incompatible changes that are currently in master. If you want to help, it would be best to help in moving the code to `iterm2`. I haven't yet had the time to do it myself.
